### ISSUE TITLE
Clickhouse support ?

### ISSUE THREAD

**Issue by kingslyChen** [zhenghaoz/gorse#211]
Any chance for clickhouse  database support?

**Comment by zhenghaoz** [comment]
Duplicate of #184

**Comment by zhenghaoz** [comment]
ClickHouse is a little different from relational database. Its interface is harder to implement. But we could support it in the future.

**Comment by kingslyChen** [comment]
> ClickHouse is a little different from relational database. Its interface is harder to implement. But we could support it in the future.

Expecting

**Comment by gyf19** [comment]
+1

**Comment by zhenghaoz** [comment]
Unfortunately, the process of ClickHouse support is blocked by mailru/go-clickhouse#133.
